pkg/cmd/server/api/types.go

```diff
@@ -301,6 +304,9 @@ type OpenIDIdentityProvider struct {
 	// ExtraScopes are any scopes to request in addition to the standard "openid" scope.
 	ExtraScopes []string
 
+	// ExtraAuthorizeParameters are any custom parameters to add to the authorize request.
+	ExtraAuthorizeParameters map[string]string
```

Comment:
What utility does this have for someone writing config?  I don't remember us stuffing all the claims into `.Extra` or anything, though I'm not necessarily opposed.  I would worry about staleness though.